src/ui/pages/ManageWallets/WalletGroup/WalletGroup.tsx

```diff
@@ -33,13 +33,29 @@ import { WalletAvatar } from 'src/ui/components/WalletAvatar';
 import { InputDecorator } from 'src/ui/ui-kit/Input/InputDecorator';
 import { BackupInfoNote } from 'src/ui/components/BackupInfoNote';
 import {
+  ContainerType,
+  getContainerType,
   isHardwareContainer,
   isMnemonicContainer,
-  isPrivateKeyContainer,
   isSignerContainer,
 } from 'src/shared/types/validators';
 import { NeutralDecimals } from 'src/ui/ui-kit/NeutralDecimals';
 
+const strings = {
+  recoveryPhraseTitle: 'Recovery Phrase',
+  privateKeyTitle: 'Private Key',
+  removeWalletSubtitle: (containerType: ContainerType) =>
```

Comment:
Meybe we can rename this property somehow. Looks like a method to remove something :))
What about `getRemoveSubtitle`?

Reply:
Oh, good point!!